Modules/Core/Common/include/itkConnectedImageNeighborhoodShape.h

```diff
@@ -299,7 +297,15 @@ GenerateConnectedImageNeighborhoodShapeOffsets() ITK_NOEXCEPT
   return offsets;
 }
 
+
+#if !defined(ITK_LEGACY_REMOVE)
+namespace Experimental // For ITK 5.0 legacy support
```

Comment:
Are any remote modules using `Experimental` namespace? If not, we don't need this. By definition, experimental things are subject to change, and my expectation is without backwards compatibility.

Reply:
@dzenanz 
> Are any remote modules using Experimental namespace? 

Honestly, I don't know. 

I really hope that some ITK users are already using things from `Experimental`, but again, I don't know. The projects I'm involved with only _indirectly_ make use of them, via `itk::GaussianDerivativeImageFunction`, `itk::N4BiasFieldCorrectionImageFilter`, and `itk::ImageMaskSpatialObject` (which do now internally use `Experimental` things).

Reply:
@dzenanz You wrote on Oct 16, 2019 https://github.com/InsightSoftwareConsortium/ITK/pull/1316/commits/27b703931bbf730ae3945cca22ec7adaffd2a9ba#r335469363 
>Are any remote modules using Experimental namespace? If not, we don't need this

Do you still think so? (Just double-checking.) Then I will make sure that the PR will no longer add `#if !defined(ITK_LEGACY_REMOVE)`  ITK 5.0 backwards compatibility.

Reply:
Number of remotes to use this is probably 0-2 (skewed towards zero). Removing `Experimental::` prefix should be easy to do, and easy to deduce as the solution in the case the problem arises.